tests/integration-tests/tests/update/test_update.py

```diff
@@ -73,7 +81,11 @@ def test_update_slurm(region, pcluster_config_reader, s3_bucket_factory, cluster
         "queue2": {
             "compute_resources": {
                 "queue2-i1": {
-                    "instance_type": "c5n.18xlarge",
+                    "instance_type_list": [
+                        {
+                            "instance_type": "c5n.18xlarge",
+                        }
+                    ],
```

Comment:
We can be consistent and use `InstanceTypeList` in the updated queue as well: [updated_queues_config](https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/tests/update/test_update.py#L116-L116).

Reply:
done